src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -6270,35 +6270,37 @@ private static string GetNamespaceToRemove(CompletionContext context, TypeComple
         internal static List<CompletionResult> CompleteHelpTopics(CompletionContext context)
         {
             var results = new List<CompletionResult>();
-            var searchPaths = new List<string>();
-            var currentCulture = CultureInfo.CurrentCulture.Name;
-
-            // Add the user scope path first, since it is searched in order.
-            var userHelpRoot = Path.Combine(HelpUtils.GetUserHomeHelpSearchPath(), currentCulture);
-
-            if (Directory.Exists(userHelpRoot))
-            {
-                searchPaths.Add(userHelpRoot);
-            }
-
-            var dirPath = Path.Combine(Utils.GetApplicationBase(Utils.DefaultPowerShellShellID), currentCulture);
-            searchPaths.Add(dirPath);
-
-            var wordToComplete = context.WordToComplete + "*";
-            var topicPattern = WildcardPattern.Get("about_*.help.txt", WildcardOptions.IgnoreCase);
-            List<string> files = new List<string>();
+            string userHelpDir = HelpUtils.GetUserHomeHelpSearchPath();
+            string appHelpDir = Utils.GetApplicationBase(Utils.DefaultPowerShellShellID);
+            string currentCulture = CultureInfo.CurrentCulture.Name;
+
+            //search for help files for the current culture + en-US as fallback
+            var searchPaths = new string[4]
+            { 
+                Path.Combine(userHelpDir, currentCulture),
+                Path.Combine(userHelpDir, "en-US"),
+                Path.Combine(appHelpDir, currentCulture),
+                Path.Combine(appHelpDir, "en-US")
+            }.Distinct();
+
+            string wordToComplete = context.WordToComplete + "*";
+            var foundTopics = new SortedSet<string>(StringComparer.OrdinalIgnoreCase);
 
             try
             {
                 var wildcardPattern = WildcardPattern.Get(wordToComplete, WildcardOptions.IgnoreCase);
 
                 foreach (var dir in searchPaths)
                 {
-                    foreach (var file in Directory.EnumerateFiles(dir))
+                    if (Directory.Exists(dir))
                     {
-                        if (wildcardPattern.IsMatch(Path.GetFileName(file)))
+                        var currentDir = new DirectoryInfo(dir);
+                        foreach (var file in currentDir.EnumerateFiles("about_*.help.txt"))
```

Comment:
If you are already using DirectoryInfo:
```c#
                    var currentDir = new DirectoryInfo(dir);
                    if (currentDir.Exists)
                    {
                        foreach (var file in currentDir.EnumerateFiles("about_*.help.txt"))
```

Reply:
Good idea, I wrongly assumed that new DirectoryInfo would throw an error if the folder didn't exist.